integreat_cms/summ_ai_api/utils.py

```diff
@@ -0,0 +1,323 @@
+        self.text = unescape(self.segment.text_content())
+
+    def translate(self, translated_text):
+        """
+        Translate the current HTML segment and create new sub elements for line breaks
+
+        :param translated_text: The translated text
+        :type translated_text: str
+        """
+        # Split the text by newlines characters
+        lines = translated_text.splitlines()
+        # Take the first line as initial text
+        self.segment.text = lines[0]
+        # If there are more than one line returned, insert <br> tags
+        for line in lines[1:]:
+            SubElement(self.segment, "br").tail = line
+
+
+# pylint: disable=too-few-public-methods
+class HTMLField:
+    """
+    A class for more complex HTML fields which are splitted into segments
+    """
+
+    #: The list of HTML segments
+    segments = []
+    #: The current HTML stream
+    html = None
+
+    def __init__(self, html):
+        """
+        Parse the HTML string into an lxml tree object and split into segments
+
+        :param html: The HTML string content of this field
+        :type html: str
+        """
+        if html:
+            self.html = fromstring(html)
+            # Translate all <p> and <li> tags (and filter out empty segments)
+            self.segments = [
+                HTMLSegment(segment=segment) for segment in self.html.iter("p", "li")
+            ]
+
+    def __repr__(self):
+        """
+        The representation used for logging
+
+        :return: The canonical string representation of the HTML field
+        :rtype: str
+        """
+        return f"<HTMLField (segments: {self.segments})>"
+
+    @property
+    def translated_text(self):
+        """
+        Assemble the content of the HTML segments into a HTML string again
+
+        :returns: The translated HTML
+        :rtype: str
+        """
+        if self.html is not None:
+            return tostring(
+                self.html, encoding="unicode", method="html", pretty_print=True
+            )
+        return None
+
+    @property
+    def exception(self):
+        """
+        Check if any of the segments experienced an error
+
+        :returns: The first exception of this HTML field
+        :rtype: Exception
+        """
+        return next(
+            (segment.exception for segment in self.segments if segment.exception), None
+        )
+
+
+# pylint: disable=too-many-instance-attributes
+class TranslationHelper:
+    """
+    Custom helper class for interaction with SUMM.AI
+    """
+
+    #: The current request
+    request = None
+    #: The form class of the current content type
+    form_class = None
+    #: The current object instance to be translated
+    object_instance = None
+    #: The German source translation of the object instance
+    german_translation = None
+    #: Wether or not the translation was successfull
+    valid = True
+    #: The title field of this helper
+    title_field = None
```

Comment:
Can you remove the title field from the automatic translation and always keep the german one? So far, the API seems to work rather bad with short texts (wrong spelling, super long translations and so on). This is also communicated with the service team

Reply:
Alright! Should I just disable it temporarily, so we might be able to translate it in the future, or should I simplify the process and remove all title functionality altogether?

Reply:
I would remove the functionality in total. Right now the code for it only creates some extra complexity and we could still take your code from the commit history of this repo if we would need in the end :)

Reply:
Due to my desire to force-push, this would not be preserved in the git history :sweat_smile: 
But I found a way to make this a bit more generic without hardcoding the field names, so we can adapt them in the future.